### PR TITLE
:recycle: Refactor front

### PR DESCRIPTION
**Pull request by renevds**
Logout menu fixes
Fix garbage colours
Closes part of #349 
Add map to tours data page
Add sorting to several parts of the data management

### CODE REVIEW

**Review by JasperJanin — DISMISSED** [review]

**Comment by JasperJanin** [comment]
@renevds Maybe change the switch button on student view to "naar beheer"
image

Still approved because original had this as well

**Review by milachae — CHANGES_REQUESTED** [review]
remove the console.logs

File: frontend/src/components/forms/forms-components/TourBuildingAdd.jsx [hunk 1]
```diff
@@ -122,9 +123,23 @@ export default function TourBuildingAdd({ tourId, callback }) {
...
 
+  console.log(selectedBuildings);
+
+  const route = selectedBuildings.map(
+    (selected) =>
+      `${selected.building.address_line_1} ${selected.building.address_line_2}`
+  );
+
+  console.log(route);
+
```
Comment (milachae) [inline frontend/src/components/forms/forms-components/TourBuildingAdd.jsx:134]: Remove the console.log()

**Review by milachae — CHANGES_REQUESTED** [review]
Most of the things are working!
- The navbar features seems to work fine.
- The sorting is also working
- I was not able to test the mapview, because I have no up to date API key

How does the mapview behave when no buildings are present? I was not able to test this.

Remove some console.log()

**Comment by renevds** [comment]
With no buildings, it just shows an empty map.

**Review by milachae — APPROVED** [review]

**Pull request merged**